sdcm/cluster.py

```diff
@@ -3733,6 +3734,7 @@ def execute_cmd(cql_session, entity_type):
                     continue
 
                 if is_column_type and (filter_out_table_with_counter and "counter" in row.type):
+                    counter_tables.add(table_name)
```

Comment:
why does this work for system and cdc tables but not counters?

Reply:
Don't know why it does work for system or CDC.
One counter table in this case contains two columns:
Row(keyspace_name='scylla_bench', table_name='test_counters', type='bigint'), type: bigint
Row(keyspace_name='scylla_bench', table_name='test_counters', type='counter'), type: counter

The first one was the reason why `test_counters` table was not filtered.

Reply:
With this schema:
```
CREATE TABLE scylla_bench.test_counters (
    pk bigint,
    ck bigint,
    c1 counter,
    c2 counter,
    c3 counter,
    c4 counter,
    c5 counter,
    PRIMARY KEY (pk, ck)
    ```